packages/terra-time-input/src/TimeInput.scss

```diff
@@ -1,40 +1,212 @@
+    height: var(--terra-time-input-height, 1.857rem);
+    line-height: var(--terra-time-input-line-height, 1.429);
+    min-width: var(--terra-time-input-width-input, 3.57em);
+    padding-bottom: var(--terra-time-input-input-padding-bottom, 0.357em);
+    padding-left: var(--terra-time-input-input-padding-bottom, 0.357em);
+    padding-right: var(--terra-time-input-input-padding-top, 0.357em);
+    padding-top: var(--terra-time-input-input-padding-top, 0.357em);
+    text-align: center;
+    transition-duration: var(--terra-time-input-transition-duration, 0.2s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-time-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:focus {
+      @include focus-styles;
+
+      background-size: var(--terra-time-input-focus-keyboard-animation-size, 0 0);
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+      }
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
+      }
+    }
+
+    &.desktop {
+      background-size: 0 0;
+      border-color: transparent;
       display: inline-block;
-      height: 1.524em;
-      margin: 0;
-      min-width: 2.1em;
-      text-align: center;
-      width: 2.1em;
+      height: 100%;
+      min-width: 1.9em;
+      padding: 0;
+      width: 1.9em;
 
       &:focus {
-        border-color: none;
-        outline: none;
+        @include focus-styles;
+
+        background-size: 0 0;
+
+        /* stylelint-disable max-nesting-depth */
+        [dir=ltr] & {
+          box-shadow: var(--terra-time-input-desktop-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+        }
+
+        [dir=rtl] & {
+          box-shadow: var(--terra-time-input-desktop-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
+        }
+        /* stylelint-enable max-nesting-depth */
       }
     }
   }
+
+  .time-input-minute.desktop {
+    min-width: 2.3em;
+    width: 2.3em;
+  }
+
+  .time-spacer {
+    line-height: var(--terra-time-input-line-height, 1.429);
+    margin-bottom: var(--terra-time-input-desktop-input-padding-bottom, 0.357em);
+    margin-top: var(--terra-time-input-desktop-input-padding-top, 0.357em);
+  }
+
+  .mobile-time-picker {
+    display: flex;
+  }
+
+  .mobile-input-label {
+    color: var(--terra-time-input-mobile-label-color, rgba(0, 0, 0, 0.5));
+    font-size: var(--terra-time-input-mobile-label-size, 0.857rem);
+  }
+
+  .meridiem-display {
+    border-color: transparent;
+    border-radius: var(--terra-time-input-border-radius, 3px);
+    border-style: var(--terra-time-input-keyboard-border-style, solid);
+    border-width: var(--terra-time-input-keyboard-border-width, 1px);
+    display: inline-block;
+    line-height: var(--terra-time-input-meridiem-display-line-height, 1.429);
+    margin-left: var(--terra-time-input-meridiem-display-left-margin, 0.357em);
+
```

Comment:
The focus box for the `meridiem-display` has different spacing than the input value's focus box - if that isn't your intention, adding some padding should put some space.

Reply:
Try it now. I just pushed new styles up. https://terra-core-deployed-pr-1027.herokuapp.com/

Reply:
Hmm looks like it's still off (currently using chrome):
![screen shot 2017-11-30 at 3 33 43 pm](https://user-images.githubusercontent.com/14063284/33456138-167a1980-d5e4-11e7-8682-3f32ed4d6921.png)
![screen shot 2017-11-30 at 3 33 56 pm](https://user-images.githubusercontent.com/14063284/33456148-1a8862b6-d5e4-11e7-8aca-57131a3b2248.png)

Reply:
Thats why. I didn't push it like I thought I did :(

Reply:
Should the default values add some padding in the first place?

Reply:
Paddings are removed. As for the other styles with padding, we do need them so that the components are able to be aligned and sized correctly.